packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -204,24 +204,25 @@ export class GraphQLTransform {
   }: TransformOption): void {
     this.seenTransformations = {};
     const parsedDocument = parse(schema);
-    const context = new TransformerContext(
-      scope,
+    const context = new TransformerContext({
```

Comment:
Converted TransformerContext arguments to take an named option object instead of a list -- it was getting too confusing to figure out what argument was in which position.